webpack/dev.server.js

```diff
@@ -42,18 +45,38 @@ module.exports = WebpackCommon({
     port: 3000,
     host: '0.0.0.0',
   },
+  devtool: 'eval-source-map',
   plugins: [
-    new webpack.SourceMapDevToolPlugin({
-      filename: '[file].map',
-      exclude: /vendors.*.*/,
-    }),
-    new webpack.HotModuleReplacementPlugin(),
     new CircularDependencyPlugin({
       exclude: /a\.js|node_modules/,
       include: /jsapp/,
       failOnError: false,
       allowAsyncCycles: false,
       cwd: process.cwd(),
     }),
+    // Some other plugins; see below.
   ],
 });
+
+// Print speed measurements if env variable MEASURE is set
+// Be careful with this, as SpeedMeasurePlugin's wrap(config) sometimes prevents
+// other plugins from working correctly - particularly in Dev mode.
+// Usually the workaround is to wrap first, then add the plugins that don't
+// enjoy being wrapped.
+// - https://github.com/stephencookdev/speed-measure-webpack-plugin/issues/160
+// - https://github.com/stephencookdev/speed-measure-webpack-plugin/issues/167
+// - https://github.com/stephencookdev/speed-measure-webpack-plugin/issues/175
+if (process.env.MEASURE) {
```

Comment:
Let's give this environment variable a more specific (and verbose and harder to type, sorry) name like `MEASURE_WEBPACK_PLUGIN_SPEED`